src/librustdoc/html/templates/page.html

```diff
@@ -13,13 +13,23 @@
           href="{{static_root_path | safe}}rustdoc{{page.resource_suffix}}.css" {# -#}
           id="mainThemeStyle"> {#- -#}
     {{- style_files | safe -}}
-    <script id="default-settings" {# -#}
+    <script defer id="default-settings" {# -#}
```

Comment:
Let's omit the `defer` here. This doesn't actually load any external JS, and is just a convenient place to hang some data- attributes.

Reply:
Oh and there's even a citation from MDN: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-defer

> defer
> Warning: This attribute must not be used if the src attribute is absent (i.e. for inline scripts), in this case it would have no effect.